worker/meson.build

```diff
@@ -44,6 +44,18 @@ if get_option('ms_rtc_logger_rtp')
   ]
 endif
 
+cpp = meson.get_compiler('cpp')
+
+# This is a workaround to define FLATBUFFERS_LOCALE_INDEPENDENT=0 outside
+# flatbuffers project, which is needed in case the current arch doesn't support
+# strtoll_l or strtoull_l (such as musl in Alpine Linux). Problem is that
+# flatbuffers build system not only defines it for its own usage but also relies
+# on it in its include/flatbuffers/util.h file, and if such a file is included
+# by mediasoup source files (and it is included) build fails with "error:
+# 'strtoull_l' was not declared in this scope".
+# See issue: https://github.com/versatica/mediasoup/issues/1223
+add_project_arguments('-DFLATBUFFERS_LOCALE_INDEPENDENT=@0@'.format(cpp.has_function('strtoull_l')), language: 'cpp')
```

Comment:
NOTE: this is copy paste from the meson.build in flatbuffers.wrap file.

Reply:
Is this something we should report upstream maybe? Having flatbuffers depend on locale is odd.